src/debug-agent.cc

```diff
@@ -148,7 +148,8 @@ void Agent::Stop() {
   err = uv_thread_join(&thread_);
   CHECK_EQ(err, 0);
 
-  uv_close(reinterpret_cast<uv_handle_t*>(&child_signal_), nullptr);
+  uv_walk(&child_loop_, WalkCloseCb, nullptr);
```

Comment:
You don't need to add a method for this, you can just use a lambda function (or even `reinterpret_cast<uv_walk_cb>(uv_close)`, although that's a bit of a hack.)